org.eclipse.transformer.parent/pom.xml

```diff
@@ -25,8 +25,8 @@
 		<revision>0.6.0-SNAPSHOT</revision>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<java.version>1.8</java.version>
-		<java.release>8</java.release>
+		<java.version>11</java.version>
```

Comment:
I am not sure this is a good thing without some agreement from other users. I have no idea if anyone depends on this working for Java 8 which is the base Java level for Jakarta EE 9.

Reply:
See comment above. That's the reason why using some basic maths to convert the duration was enough in this case. I can revert this change and move back the toHoursMinutesSeconds() method above.